src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -395,6 +395,200 @@ const POLYGON_INPUT = new GraphQLList(new GraphQLNonNull(GEO_POINT_INPUT));
 
 const POLYGON = new GraphQLList(new GraphQLNonNull(GEO_POINT));
 
+const USER_ACL_INPUT = new GraphQLInputObjectType({
+  name: 'UserACLInput',
+  description:
+    'Allow to manage users in ACL. If read and write are not provided the user will have read and write rights.',
+  fields: {
+    userId: {
+      description: 'ID of the targetted User.',
+      type: new GraphQLNonNull(GraphQLID),
+    },
+    read: {
+      description: 'Allow the user to read the current object.',
+      type: GraphQLBoolean,
+    },
+    write: {
+      description:
+        'Allow the user to write on the current object. If true, read will be set to true.',
+      type: GraphQLBoolean,
+    },
+  },
+});
+
+const ROLE_ACL_INPUT = new GraphQLInputObjectType({
+  name: 'RoleACLInput',
+  description:
+    'Allow to manage roles in ACL. If read and write are not provided the role will have read and write rights.',
+  fields: {
+    roleName: {
+      description: 'Name of the targetted Role.',
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    read: {
+      description:
+        'Read is true by default. Allow users who are members of the role to read the current object.',
+      type: GraphQLBoolean,
+    },
+    write: {
+      description:
+        'Write is true by default. Allow users who are members of the role to write on the current object. If true, read will be set to true.',
+      type: GraphQLBoolean,
+    },
+  },
+});
+
+const PUBLIC_ACL_INPUT = new GraphQLInputObjectType({
+  name: 'PublicACLInput',
+  description: 'Allow to manage public rights.',
+  fields: {
+    read: {
+      description:
+        'Read is true by default. Allow anyone to read the current object.',
+      type: GraphQLBoolean,
+    },
+    write: {
+      description:
+        'Write is true by default. Allow anyone to write on the current object. If true, read will be set to true.',
```

Comment:
Do we really need to say "If true, read will be set to true.", since read is true by default?

Reply:
Yes, i think here the information is important, we need to be as clear as possible about the behavior of ACL.

Reply:
Do you mean that we cannot have read=false and write=true?